src/kudu/consensus/consensus_peers.cc

```diff
@@ -98,6 +98,9 @@ TAG_FLAG(raft_proxy_max_hops, advanced);
 
 DECLARE_int32(raft_heartbeat_interval_ms);
 
+DEFINE_int32(proxy_batch_duration_ms, 500,
```

Comment:
500ms is heartbeat value today. Wondering if we only want to send 1 rpc
per HB or more. if write rate is 4MBPs, 500ms will be too low, since batch
size is 2MB

Reply:
We can come up with a good default on the fbcode side